HousingRegisterApi.Tests/V1/Services/BedroomCalculationServiceTests.cs

```diff
@@ -0,0 +1,510 @@
+using HousingRegisterApi.V1.Domain;
+using HousingRegisterApi.V1.Services;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace HousingRegisterApi.Tests.V1.Services
+{
+    [TestFixture]
+    public class BedroomCalculationServiceTests
+    {
+        private const string Male = "male";
+        private const string Female = "female";
+        private const string MainApplicant = "";
+        private const string MainApplicantIsPartner = "partner";
+        private const string MainApplicantIsMyParent = "parent";
+        //private const string MainApplicantIsMyBrother = "brother";
```

Comment:
Remove if not relevant/used.

Reply:
I wanted to add some more tests around this, but, will remove